k8s/helm-chart/dynatrace-gcp-monitor/values.yaml

```diff
@@ -172,12 +172,14 @@ clusterName: ""
 # scopingProjectSupportEnabled: set to true when metrics scope is configured, so metrics will be collected from all monitored projects.
 scopingProjectSupportEnabled: "false"
 # excludedProjects: comma separated list of projects that will be excluded from monitoring (e.g. "project-a,project-b,project-c").
+# This parameter is only applicable to Standard deployments (with other projects metrics scope disabled)
 excludedProjects: ""
 # excludedProjectsByPrefix: comma separated list of projects substring that will be excluded from monitoring (e.g. "project-a,proj,pro").
+# This parameter is only applicable to Standard deployments (with other projects metrics scope disabled)
 excludedProjectsByPrefix: ""
-# includedProjects: comma separated list of projects that will be included from monitoring (e.g. "project-a,project-b,project-c").
+# includedProjects: comma separated list of projects that will be included in monitoring (e.g. "project-a,project-b,project-c").
```

Comment:
@Dawid-Kaszubski is similar limitation applicable to `includeprojects` option? do we also need addiytional comment to be added here?

Reply:
You're right. A similar limitation is applicable to the `includeprojects` option. It'd be great to have an additional comment added here too.